proto/go_proto_library.bzl

```diff
@@ -300,45 +320,45 @@ _well_known_imports = [
     "wrappers",
 ]
 
+
 # If you have well_known proto deps, rules_go will add a magic
 # google/protobuf/ directory at the import root
 def go_google_protobuf(name = _GO_GOOGLE_PROTOBUF):
-  deps = [_add_target_suffix(_well_known_import_key(wk), _PROTOS_SUFFIX)
-          for wk in _well_known_imports]
+  deps = [
+      _add_target_suffix(_well_known_import_key(wk), _PROTOS_SUFFIX)
+      for wk in _well_known_imports
+  ]
   outs = [wk + ".proto" for wk in _well_known_imports]
 
   native.genrule(
       name = name,
       srcs = deps,
-      outs = ["google/protobuf/"+o for o in outs],
+      outs = ["google/protobuf/" + o for o in outs],
       cmd = "cp $(SRCS) $(@D)/google/protobuf/",
-      visibility = ["//visibility:public"],
-  )
+      visibility = ["//visibility:public"])
+
 
 def go_proto_repositories(shared = 1):
   """Add this to your WORKSPACE to pull in all of the needed dependencies."""
   new_go_repository(
       name = "com_github_golang_protobuf",
       importpath = "github.com/golang/protobuf",
-      commit = "8ee79997227bf9b34611aee7946ae64735e6fd93",
-  )
+      commit = "8ee79997227bf9b34611aee7946ae64735e6fd93")
   if shared:
     # if using multiple *_proto_library, allows caller to skip this.
     native.http_archive(
         name = "com_github_google_protobuf",
         url = "https://github.com/google/protobuf/archive/v3.2.0.tar.gz",
         strip_prefix = "protobuf-3.2.0",
-        sha256 = "2a25c2b71c707c5552ec9afdfb22532a93a339e1ca5d38f163fe4107af08c54c",
-    )
+        sha256 =
+        "2a25c2b71c707c5552ec9afdfb22532a93a339e1ca5d38f163fe4107af08c54c")
 
   # Needed for gRPC, only loaded by bazel if used
   new_go_repository(
       name = "org_golang_x_net",
       commit = "4971afdc2f162e82d185353533d3cf16188a9f4e",
-      importpath = "golang.org/x/net",
-  )
+      importpath = "golang.org/x/net")
   new_go_repository(
       name = "org_golang_google_grpc",
       tag = "v1.0.4",
-      importpath = "google.golang.org/grpc",
```

Comment:
should stay like before to match BUILD syntax of trailing ',' and ')' on next line